app/views/problem_reports/index.html.erb

```diff
@@ -0,0 +1,63 @@
+<h3> Problem Report List (<%= params[:filter] %>) </h3>
+
+<% if flash[:notice] %>
```

Comment:
May I suggest something like this -

1. Separate this into `flash[:notice]` and `flash[:alert]`
2. Print the contents of the flash directly
3. Just do `if flash` instead